.github/workflows/e2e.yml

```diff
@@ -0,0 +1,28 @@
+name: Scheduled E2E
```

Comment:
need a different name .. this one is used to post the results of smoke below -> Slack -support channel
we should do the same for this one -> -dev channel (@Dev-Alex23 can show you)